oci-test/tests/oci_resourcemanager_stack/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "resource_name" {
+  type        = string
+  default     = "steampipetest20200125"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "tenancy_ocid" {
+  type        = string
+  default     = ""
+  description = "OCI credentials profile used for the test. Default is to use the default profile."
+}
+
+variable "config_file_profile" {
+  type        = string
+  default     = "DEFAULT"
+  description = "OCI credentials profile used for the test. Default is to use the default profile."
+}
+
+variable "region" {
+  type        = string
+  default     = "ap-mumbai-1"
+  description = "OCI region used for the test. Does not work with default region in config, so must be defined here."
+}
+
+variable "template_id" {
+  type        = string
+  default     = "ocid1.ormtemplate.oc1.ap-mumbai-1.aaaaaaaa2ereypft5pb3vgmplr6arm767dpqvkfocugw4vdcbrgmmg2dbcsq"
```

Comment:
can we make it dynamic Instead of the hard coded value? if some one will delete this default template then the intg test will fail.